### PR TITLE
#161966603 Api documentation

### PR DESCRIPTION
**Pull request by salma-nyagaka**
#### What does this PR do?
Document API using swagger
#### Description of Task to be completed?
Configure the application for Swagger documentation
#### How should this be manually tested?
- git clone https://github.com/andela/ah-technocrats.git
- cd ah-technocrats
- git checkout-api-documentation-161966603
#### Any background context you want to provide?
This is required to provide instructions about how to effectively use the API and contains all the information required to work with the API
#### What are the relevant pivotal tracker stories?
[#161966603](https://www.pivotaltracker.com/n/projects/2225606)

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] At least 2 people have reviewed my PR
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] My PR has one commit.

### CODE REVIEW

**Review by verenceLola — COMMENTED** [review]
Nice working with you

**Review by SilasKenneth — COMMENTED** [review]
Good work

**Review by SilasKenneth — DISMISSED** [review]

**Comment by TeamoreA** [comment]
Nice work though your checks are failing

**Review by SilasKenneth — DISMISSED** [review]

**Review by SilasKenneth — DISMISSED** [review]

**Review by Mnickii — CHANGES_REQUESTED** [review]
Rebase and fix errors

**Review by SilasKenneth — DISMISSED** [review]

**Review by SilasKenneth — DISMISSED** [review]

**Comment by SnyderMbishai** [comment]
Good work @salma-nyagaka

**Review by SnyderMbishai — DISMISSED** [review]

**Comment by kipropbrian** [comment]
There is something wrong with this PR. Commits from other PR's are appearing here?

File: README.md [hunk 1]
```diff
@@ -1,20 +1,41 @@
 ![license](https://img.shields.io/github/license/mashape/apistatus.svg)
+[![Coverage Status](https://coveralls.io/repos/github/andela/ah-technocrats/badge.svg?branch=develop)](https://coveralls.io/github/andela/ah-technocrats?branch=develop)
```
Comment (Mnickii) [inline README.md]: This is not related to the story referenced in this PR

File: authors/apps/authentication/tests/test_login.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+class TestAuthentication(BaseTestCase):
+    """Test class for the authentication"""
+
+
+    def test_user_login(self):
+        """test login with accurate data"""
+        self.client.post(
+            self.register_url,
+            self.register_data,
```
Comment (Mnickii) [inline authors/apps/authentication/tests/test_login.py]: Neither is this

**Review by Mnickii — CHANGES_REQUESTED** [review]
Alot of your changes are unrelated to adding swagger docs. Make sure only changes pertaining to this story are present in this PR

**Comment by actlikewill** [comment]
Some check are not passing...

File: README.md (hunk 1 above)
Comment (verenceLola) [reply README.md]: We've worked on your feedback.

File: authors/apps/authentication/tests/test_login.py (hunk 2 above)
Comment (verenceLola) [reply authors/apps/authentication/tests/test_login.py]: Fixed it.

**Review by Mnickii — APPROVED** [review]
